plugin/src/main/java/io/jenkins/plugins/casc/model/Mapping.java

```diff
@@ -47,7 +47,15 @@ public void putIfNotNull(String key, CNode node) {
         if (node != null) super.put(key, node);
     }
 
+    /**
+     * @deprecated Misspelled name, use {@link #putIfNotEmpty(String, Sequence)} instead.
+     */
+    @Deprecated
     public void putIfNotEmpry(String key, Sequence seq) {
```

Comment:
should change any internal plugin calls to use the correct version

Reply:
If there are, +1

Reply:
Apparently there aren't any, just checked

The only reference to it I could find on GitHub is in the `cloudbees-jenkins-advisor-plugin` in its tests, I think we could probably just remove the method?

Reply:
if no other method is referenced on GitHub I feel safe enough removing putIfNotEmpry 😄

Reply:
Which is obviously how I found the issue :-)
Advisor isn't yet released with the JCasC integration.. Let's remove it for now I will replace by a condition on my side